mrp_bom_cust_analytics_valuation/views/mrp_unbuild_process_type_views.xml

```diff
@@ -0,0 +1,38 @@
+<odoo>
+    <data>
+
+        <!-- Form View -->
+        <record id="mrp_bom_form_process_type_form_view_mrp_bom_cust_analytics_valuation" model="ir.ui.view">
+            <field name="name">mrp.unbuild.process.type.view.form (mrp_bom_cust_analytics_valuation)</field>
+            <field name="model">mrp.unbuild.process.type</field>
+            <field name="inherit_id" ref="mrp_unbuild_cust_alu_analytics.view_mrp_process_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='cmplanta_id']" position="after">
+                    <field name="labour" class="oe_inline"/> per hour
+                    <field name="energy" class="oe_inline"/> per hour
+                    <field name="amortization" class="oe_inline"/> per hour
+                    <field name="repair_maintenance_mgmt" class="oe_inline"/> per hour
+                    <field name="currency_id" invisible="1"/>
+                </xpath>
+            </field>
+        </record>
+
+        <!-- Tree View -->
+        <record id="mrp_bom_tree_process_type_tree_view_mrp_bom_cust_analytics_valuation" model="ir.ui.view">
+            <field name="name">mrp.unbuild.process.type.view.tree (mrp_bom_cust_analytics_valuation)</field>
+            <field name="model">mrp.unbuild.process.type</field>
+            <field name="inherit_id" ref="mrp_unbuild_cust_alu_analytics.view_mrp_process_tree"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='cmplanta_id']" position="after">
```

Comment:
> Could you provide a preview screenshot of the view?

Still missing this 👆

Reply:
Now it depends on company_id instead of cmplanta_id
Before:
![image](https://github.com/solvosci/slv-manufacture/assets/32322359/ba58d52f-4401-445b-8295-49c8ea2d46d3)

After:
![image](https://github.com/solvosci/slv-manufacture/assets/32322359/2766e6e7-94e6-48a4-8d8d-6e20d3928fc9)